gr-blocks/lib/add_blk_impl.cc

```diff
@@ -34,25 +57,19 @@ add_blk_impl<float>::add_blk_impl(size_t vlen)
                  io_signature::make(1, 1, sizeof(float) * vlen)),
       d_vlen(vlen)
 {
-    const int alignment_multiple = volk_get_alignment() / sizeof(float);
-    set_alignment(std::max(1, alignment_multiple));
+    set_alignment(std::max(1, int(volk_get_alignment() / sizeof(float))));
 }
 
 template <>
-int add_blk_impl<float>::work(int noutput_items,
-                              gr_vector_const_void_star& input_items,
-                              gr_vector_void_star& output_items)
+add_blk_impl<gr_complex>::add_blk_impl(size_t vlen)
```

Comment:
Just wondering: Is this specialization of the constructor (and the one for `float`) required? Was it ever required? (As per C++ rules – my knowledge of templates is limited…)